Example/ExampleAPI/Endpoint/FileUploadEndpoint.swift

```diff
@@ -10,23 +10,30 @@ import Apexy
 
 /// Endpoint for uploading a file
 public struct FileUploadEndpoint: UploadEndpoint {
-    
+
     public typealias Content = Void
+    public typealias Failure = Error
     
     private let fileURL: URL
     
     public init(fileURL: URL) {
         self.fileURL = fileURL
     }
     
-    public func content(from response: URLResponse?, with body: Data) throws {
-        try ResponseValidator.validate(response, with: body)
-    }
-    
-    public func makeRequest() throws -> (URLRequest, UploadEndpointBody) {
+    public func makeRequest() -> Result<(URLRequest, UploadEndpointBody), Error> {
```

Comment:
Should the return type be `Result<(URLRequest, UploadEndpointBody), Failure>` instead of `Result<(URLRequest, UploadEndpointBody), Error>`?